transformer/common_data_provider.go

```diff
@@ -111,12 +118,35 @@ func (c CommonDataProvider) getNodeName() (string, error) {
 	return nName, nil
 }
 
+// FetchServerVersion returns the version of the Kubernetes server
+func (c CommonDataProvider) FetchServerVersion() (version.Version, error) {
+	serverVersion, err := c.kubeClient.Discovery().ServerVersion()
+	if err != nil {
+		return version.Version{}, err
+	}
+	return version.Version{
+		Version: serverVersion.Major + "." + serverVersion.Minor,
+	}, nil
+}
+
+func (c CommonDataProvider) FetchVersionInfo() (fetching.CloudbeatVersionInfo, error) {
+	serverVersion, err := c.FetchServerVersion()
+	if err != nil {
+		return fetching.CloudbeatVersionInfo{}, err
```

Comment:
Is this behavior by design?
In case we are unable to gather k8s version, do we not want to send policy and cloudbeat versions anyway?